binders/kafka-binder/spring-cloud-stream-binder-kafka-streams/src/test/java/org/springframework/cloud/stream/binder/kafka/streams/SerdeResolverUtilsTests.java

```diff
@@ -173,6 +173,15 @@ void returnsNoSerdesForWildcardType() {
 				});
 		}
 
+		@Test
+		void discardsWildcardMatchesForJavaLangObject() {
```

Comment:
I verified this fails w/o the changes to SerdeResolverUtils and passed w/ the changes.